components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/view/ImViewerComponent.java

```diff
@@ -1082,6 +1082,33 @@ public void setLookupTable(int index, String lut, boolean preview) {
     /**
      * Implemented as specified by the {@link ImViewer} interface.
      * 
+     * @see ImViewer#setReverseIntensity(int, boolean, boolean)
+     */
+    public void setReverseIntensity(int index, boolean revInt, boolean preview) {
+        switch (model.getState()) {
+        case NEW:
+        case DISCARDED:
+            throw new IllegalStateException(
+                    "This method can't be invoked in the DISCARDED, "
+                            + "NEW or LOADING_RENDERING_CONTROL state.");
```

Comment:
It's fine with me if you prefer to fix these in a separate PR, just pop it on a 5.3 card.